dt: 0.25
sim:
  steeringClamp: 45.0
  engineClamp: 1000.0
  mass: 800.0
  first_person: True
